docs/data/sdks/typescript-browser/index.md

```diff
@@ -597,16 +1219,24 @@ You need to download `plugin-page-view-tracking-browser` and add the `pageViewTr
     yarn add @amplitude/plugin-page-view-tracking-browser
     ```
 
-```ts
-import * as client from '@amplitude/analytics-browser';
-import { pageViewTrackingPlugin } from '@amplitude/plugin-page-view-tracking-browser';
+Add plugin to Ampltude instance.
```

Comment:
typo
```suggestion
Add plugin to Amplitude instance.
```

Reply:
Miss i in Amplitude.
Should be Amplitude instead of Ampltude.